src/tabs/RP_Getting_Started.tab

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomTab xmlns="http://soap.sforce.com/2006/04/metadata">
```

Comment:
as I mentioned in my first UI comments, I think we'd like to avoid another Getting Started tab in the package to avoid customer confusion.  @szeitz would you agree?